server/controllers/CommentController.js

```diff
@@ -1,5 +1,75 @@
 const db = require("../models/index");
 const Comment = db.Comment;
+const Challenge = db.Challenge;
+const Solution = db.Solution;
+const { AITA } = require("../AI/AITA");
+
+function formatComments(comments) {
+  const extracted = comments.map(
+    ({ id, text, userId, solutionId, upVotes, replies }) => ({
+      id,
+      text,
+      userId,
+      solutionId,
+      upVotes,
+      replies,
+    }),
+  );
+
+  const convertedReplies = extracted.map((c) => ({
+    ...c,
+    replies: c.replies
+      .split(",")
+      .filter((r) => r.length !== 0)
+      .map((r) => Number(r)),
+  }));
+
+  // console.log("Converted Replies", convertedReplies);
+
+  const replyIds = [];
+
+  function replaceReplies(cms, cm) {
```

Comment:
I think it would improve readability if we use `comments, comment` instead of `cms,cm`  if thats what cm stands for

Reply:
It conflicts with the variables used in the parent function.